layers/+tools/shell/funcs.el

```diff
@@ -27,6 +27,14 @@
   (let ((default-directory (projectile-project-root)))
     (call-interactively 'spacemacs/default-pop-shell)))
 
+(defun spacemacs/projectile-run-shell ()
```

Comment:
Maybe we should call it `spacemacs/default-run-shell`?

Reply:
`spacemacs/projectile-run-default-shell`